### PR TITLE
Update splice docs

### PR DESCRIPTION
**Pull request by waynecollier-da**
## Checklist for Hyperledger Lab Proposal PR

- [ ] Read [README.md](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/README.md) carefully
- [ ] Make sure your changes are committed with the proper sign-off
- [ ] Use the labs name as the PR title
- [ ] Copy the short description of the proposal as the description of the PR
- [ ] Remove this checklist section
- [ ] Submit your PR and the labs stewards will review your proposal
- [ ] Ask your sponsor to confirm sponsorship by stating so in a comment to the PR

### CODE REVIEW

File: labs/splice.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
----
-layout: default
-title: Splice
-parent: Labs
----
 # Splice
```
Comment (tkuhrt) [inline labs/splice.md:1]: The above content is needed so that the lab shows up on https://labs.hyperledger.org

File: labs/splice.md [hunk 2]
```diff
@@ -99,24 +103,14 @@ We expect that contributors interested in Splice may contribute, for example, ex
...
-# Initial Committers
-
-- https://github.com/waynecollier-da
-- https://github.com/isegall-da
-
-# Sponsor
-https://github.com/tkuhrt - Tracy Kuhrt, Chair of the Hyperledger Technical Oversight Committee
-
-# Pre-existing repository
-We have an existing private repository. We plan to copy code from that repository to the new Splice repository in the first few weeks after the project has been approved, instead of transferring the repository itself.
```
Comment (tkuhrt) [inline labs/splice.md:122]: These lines are part of the original template for lab proposals. Ideally we should probably keep these for history and to be consistent with the other lab proposals.

**Review by tkuhrt — COMMENTED** [review]
We do not need to continue to have this file updated. I am okay with the updates, but if you would rather, it might be better to just link to the lab code where the README that you have with the lab continues to be updated. Let me know your thoughts.

**Comment by waynecollier-da** [comment]
Hi, Tracy. Sorry for all the errors here.  I'l try again.

**Comment by tkuhrt** [comment]
Looks like DCO was missed on the last two commits. Please follow the [instructions](https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/268/checks?check_run_id=25660645400) to fix.

**Comment by waynecollier-da** [comment]
@tkuhrt FYI we cleaned up some of the language used in this lab description, and added a link to the Splice repo readme. After this update, we'll leave this lab description unchanged, and reference the readme for the Splice Lab repo itself.

**Comment by waynecollier-da** [comment]
@tkuhrt Are we ok to merge this now?

**Review by tkuhrt — APPROVED** [review]

**Review by alvaropicazo — APPROVED** [review]

**Pull request merged**